pyproject.toml

```diff
@@ -12,8 +12,9 @@ packages = [ { include = "volttron", from = "src" } ]
 
 [tool.poetry.dependencies]
 python = ">=3.8,<4.0"
+#volttron-lib-sql-historian="^0.1.0"
```

Comment:
This needs to be updated to not have this commented out.  Must have that lib pushed to pypi before this can be set.  It should be an alpha version.